frontend/js/collections/labels.js

```diff
@@ -58,9 +58,23 @@ define(
              */
             parse: function (resp) {
                 if (resp.labels && _.isArray(resp.labels)) {
-                    return resp.labels;
+                    // sort by abbreviation of label
+                    return resp.labels.sort((a, b) => {
```

Comment:
sorting could also be simplified with a method to remove redundant code (see categories.js)